integreat_cms/cms/templates/events/event_form.html

```diff
@@ -104,7 +104,7 @@ <h1 class="heading overflow-hidden text-ellipsis">
                                 <a id="edit-slug-btn"
                                    title="{% translate "Edit" %}"
                                    class="ml-2 btn-icon">
-                                    <i icon-name="edit-3"></i>
+                                    <i icon-name="pencil"></i>
```

Comment:
```suggestion
                                    <i icon-name="pen-line"></i>
```
`edit-3` was renamed to `pen-line`, see: https://github.com/lucide-icons/lucide/pull/1286/files#diff-bfbe0e2354e539c486b929bbff8833ed2d3da50ec61a4f2f465215d656d7111b